modules/hfscm/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/hfscm",
+	"url": "https://www.dndbeyond.com/sources/dnd/hfscm",
```

Comment:
URL is valid, redirects to `https://www.dndbeyond.com/sources/dnd/hfscm/heroes-feast-saving-the-childrens-menu`